### PR TITLE
MAT-6261 add button only enabled after required fields are filled

### PR DESCRIPTION
**Pull request by sb-cecilialiu**
## MADiE PR

Jira Ticket: [MAT-6261](https://jira.cms.gov/browse/MAT-6261)
(Optional) Related Tickets:

### All Submissions
* [x] This PR has the JIRA linked.
* [x] Required tests are included
* [x] No extemporaneous files are included (i.e Complied files or testing results)
* [x] This PR is into the **correct branch**.
* [ ] All Documentation as needed for this PR is Complete (or noted in a TODO or other Ticket)
* [ ] Any breaking changes or failing automation are noted by placing a comment on this PR.

### DevSecOps
If there is a question if this PR has a security or infrastructure impact, please contact the Security or DevOps engineer assigned to this project to discuss it further.

* [ ] This PR has NO significant security impact (i.e Changing auth methods, Adding a new user type, Adding a required but vulnerable package)
* [ ] All CDN/Web dependencies are hosted internally (i.e MADiE-Root Repo)

### Reviewers
By Approving this PR you are attesting to the following:

*  Code is maintainable and reusable, reuses existing code and infrastructure where appropriate, and accomplishes the task’s purpose
*  The tests appropriately test the new code, including edge cases
*  If you have any concerns they are brought up either to the developer assigned, security engineer, or leads

### CODE REVIEW

File: src/components/editTestCase/qdm/LeftPanel/ElementsTab/Elements/DataElementsCard/attributes/QdmEntity.tsx [hunk 1]
```diff
@@ -18,20 +18,50 @@ const QdmEntity = ({
...
+      setId(value);
+    } else if (field === "role") {
+      setRole(value);
+    } else if (field === "specialty") {
+      setSpecialty(value);
+    } else if (field === "qualification") {
+      setQualification(value);
+    } else {
+      setField(field);
+      setCode(value);
```
Comment (RohitKandimalla) [inline src/components/editTestCase/qdm/LeftPanel/ElementsTab/Elements/DataElementsCard/attributes/QdmEntity.tsx:64]: I am sorry, where is ADD button disabled ? I can't seem to find the code that add ability to enable or disable a button

Comment (sb-cecilialiu) [reply src/components/editTestCase/qdm/LeftPanel/ElementsTab/Elements/DataElementsCard/attributes/QdmEntity.tsx:64]: @RohitKandimalla The ADD button will be disabled if the attributeValue is not set. The code is in its parent component: DisplayAttributeInputs.tsx

Comment (RohitKandimalla) [reply src/components/editTestCase/qdm/LeftPanel/ElementsTab/Elements/DataElementsCard/attributes/QdmEntity.tsx:64]: Yes I looked at DisplayAttributeInputs.tsx the button doesn't have a prop disabled 

`            <Button
              tw="absolute bottom-0"
              variant="outline-filled"
              data-testid="add-attribute-button"
              onClick={handleAttributeChange}
            >
              Add
            </Button>`

Comment (sb-cecilialiu) [reply src/components/editTestCase/qdm/LeftPanel/ElementsTab/Elements/DataElementsCard/attributes/QdmEntity.tsx:64]: @RohitKandimalla No it doesn't have the disabled prop, but you can't add any attribute if the attribute value is not set

Comment (sb-cecilialiu) [reply src/components/editTestCase/qdm/LeftPanel/ElementsTab/Elements/DataElementsCard/attributes/QdmEntity.tsx:64]: @RohitKandimalla I added it anyway

**Review by RohitKandimalla — APPROVED** [review]

**Pull request merged**